src/app/services/resource.service.ts

```diff
@@ -302,7 +310,14 @@ export class ResourceService {
   getExportJobs(): Observable<any[]> {
     switch (this.endpointMode) {
       case EndpointMode.TRANSMART: {
-        return this.transmartResourceService.getExportJobs();
+        if (this.transmartResourceService.exportDataView === 'customFormat') {
```

Comment:
It seems that a check like `if (this.config['use-external-job'])` would more descriptive than this condition.

Reply:
good idea, thanks! Will change this